src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -197,6 +201,19 @@ struct patcher
   }
 };
 
+  void
```

Comment:
Hi @dezhiAmd .  The build is failing because the function is unused in release builds.
Please do this:
```suggestion
  XRT_CORE_UNUSED void
```